### PR TITLE
ci: Add primitives group to pullapprove

### PR DESCRIPTION
**Pull request by jessicajaniuk**
This adds the shared primitives reviewers groups and list of reviewers to the pullapprove rules.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] CI related changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by dylhunn — APPROVED** [review]
reviewed-for: fw-core

**Review by alxhub — APPROVED** [review]

File: .pullapprove.yml [hunk 1]
```diff
@@ -1321,6 +1321,81 @@ groups:
...
+  # External team required reviews
+  primitives-signals:
+    <<: *defaults
+    conditions:
+      - >
+        contains_any_globs(files, [
+          'packages/core/primitives/signals/**/{*,.*}',
+        ])
+    reviewers:
+      users:
```
Comment (jatraman) [inline .pullapprove.yml:1361]: can we bump the Wiz team up to own all shared primitives? that's closer to how ownership is handled on the wiz team today.

Comment (alxhub) [reply .pullapprove.yml:1361]: +1, that's reasonable - there's no reason the signals package should be special there.

**Review by jatraman — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 2ff74dcd959edf6c2cde19d5622033d5a8799ac2.

**Pull request closed without merging**